lm_eval/api/metrics.py

```diff
@@ -58,6 +58,19 @@ def f1_score(items):
     return np.max(fscore)
 
 
+@register_aggregation("squad_f1")
+def squad_f1_score(items):
+    gold_squad, pred_squad = [], []
+    for index, (ref, pred) in enumerate(items):
+        pred_dict = {'prediction_text': str(pred), 'id': str(index)}
+        ref_dict = {'answers': {'answer_start': [0], 'text': [str(ref)]}, 'id': str(index)}
+        gold_squad.append(ref_dict)
+        pred_squad.append(pred_dict)
+    squad_metric = hf_evaluate.load("squad")
+    results_squad = squad_metric.compute(predictions=pred_squad, references=gold_squad)
+    return results_squad['f1']/100
```

Comment:
It seems this is not used in favor of another function placed in `utils.py` for afrimmlu? If that's the case we can remove this.